package exceptions

object GetCEError : Throwable() {

}
